desk/app/channels-server.hoon

```diff
@@ -1269,4 +1286,74 @@
       (~(put by waiting.hooks) id.effect [origin +.effect])
     (schedule-waiting +.effect)
   ==
+++  get-hook-template
+  |=  =nest:c
+  ^-  template:h
+  =/  order=(list id-hook:h)  (~(got by order.hooks) nest)
+  =/  crons=(list [id-hook:h job:h])
+    %+  roll  ~(tap by crons.hooks)
+    |=  [[=id-hook:h =cron:h] cr=(list [id-hook:h job:h])]
+    ?~  job=(~(get by cron) nest)  cr
+    (snoc cr [id-hook u.job])
```

Comment:
Might be more clear to write as a `+murn` call, if it's just accumulating into a list.